src/Server/middleware/featureFlagMiddleware.ts

```diff
@@ -43,7 +43,14 @@ export function featureFlagMiddleware(serviceType: symbol) {
           }
         }, {})
 
+        const e2eFeatureFlags = Object.fromEntries(
+          Object.entries(featureFlags).filter(
+            ([key, value]) => key.startsWith("e2e_") && value.flagEnabled
+          )
+        )
```

Comment:
Despite what we proposed in the tech plan, [this recommends hyphenating feature names](https://www.notion.so/artsy/Feature-toggles-and-experiments-43523493b48a4a92882037f0106bef5c?pvs=4#258f640ba019472db81d3c114fd03132) so maybe we should align with that.